.github/workflows/macOS.yml

```diff
@@ -72,49 +72,35 @@ jobs:
       with:
         ruby-version: 2.7 # Not needed with a .ruby-version file
 
+    - name: Determine pre-existing brew packages
+      if: steps.dependency-cache.outputs.cache-hit != 'true'
+      run: |
+        echo 'EXISTING_CELLAR<<EOF' >> $GITHUB_ENV
+        ls -1 /usr/local/Cellar >> $GITHUB_ENV
+        echo 'EOF' >> $GITHUB_ENV
+
+    - name: Cache dependencies
+      id: dependency-cache
+      uses: actions/cache@v2
+      with:
+        path: |
+          /.npm
```

Comment:
Is this correct? Seems like `~/.npm` would be the correct path

Reply:
true, ty